File: trojan/carberp-botnet/adminpanel/bot_aa/modules/accounts/exit.php
/*
  name      Carberp Botnet
  type      trojan
  cve       —
  year      unknown
  os        Windows
  authors   unknown
  source    krisyotam
  archived  krisyotam (2026)
  notes     —
 */
<?php

$mysqli->query("update bf_users set PHPSESSID='' WHERE (id='".$_SESSION["user"]->id."') LIMIT 1");

unset($_SESSION['user']);
session_destroy();

header("Location: /");
exit;


?>

File: research/cocomelonc/2021-10-26-windows-shellcoding-1/exit.c
/*
  name      Cocomelonc
  type      research
  cve       —
  year      unknown
  os        Multi
  authors   unknown
  source    krisyotam
  archived  krisyotam (2026)
  notes     —
 */
/*
exit.c - run calc.exe and exit
*/
#include <windows.h>

int main(void) {
  WinExec("calc.exe", 0);
  ExitProcess(0);
}
